man/man8/zfs-mount-generator.8.in

```diff
@@ -45,6 +62,49 @@ history_event-zfs-list-cacher.sh .
 .RE
 .PP
 .sp
+.SS PROPERTIES
+The behavior of the generator script can be influenced by setting special properties on datasets:
+.sp
+.TP 4
+.BR org.open\-zfs.systemd:noauto
+Create and treat the mount unit as if it had
+.BR canmount=noauto
+set.
```

Comment:
I wonder if we should have a really short statement about why this is necessary.